client-lib/src/main/java/com/ibm/ws/repository/strategies/writeable/UpdateInPlaceStrategy.java

```diff
@@ -149,9 +148,8 @@ public void uploadAsset(RepositoryResourceImpl resource, List<RepositoryResource
      * @throws RepositoryBadDataException
      * @throws RepositoryResourceCreationException
      */
-    public void uploadAttachment(RepositoryResourceImpl resource, AttachmentResourceImpl attachment, RepositoryResourceImpl matchingResource)
-                    throws RepositoryResourceCreationException, RepositoryBadDataException, RepositoryResourceUpdateException,
-                    RepositoryBackendException, RepositoryResourceException {
+    public void uploadAttachment(RepositoryResourceImpl resource, AttachmentResourceImpl attachment,
```

Comment:
The exception isn't added to the javadoc here. I suspect there are other similar issues elsewhere. Don't know if the best way to fix is to remove the other exceptions or add the new one